src/pj_initcache.c

```diff
@@ -51,7 +51,7 @@ paralist *pj_clone_paralist( const paralist *list)
       newitem->used = 0;
       newitem->next = 0;
       strcpy( newitem->param, list->param );
-      
+
```

Comment:
Only non-functional changes in this file. While trailing whitespace is annoying as hell, removing it in this context only contributes clutter to the commit log of this file.

Reply:
You're obviously and evidently right - and I'm quite surprised that all functional changes that was entered along the work ended up being eradicated again (but as mentioned in the PR intro - a lot of stuff, spanning 3 continents, has been going on during this work).

Will back this out.